wikichron/dash/apps/monowiki/data_controller.py

```diff
@@ -62,25 +62,23 @@ def load_and_compute_data(wikis, metrics):
         # compute metric data:
         print(' * [Info] Starting calculations....')
         time_start_calculations = time.perf_counter()
-        data = compute_data(wikis_df, metrics)
+        metric_object_list = compute_data(wikis_df, metrics)
         time_end_calculations = time.perf_counter() - time_start_calculations
         print(' * [Timing] Calculations : {} seconds'.format(time_end_calculations) )
-        return data
+        return metric_object_list
 
 
     @cache.memoize()
-    def generate_longest_time_axis(list_of_selected_wikis, relative_time):
+    def generate_longest_time_axis(list_of_metrics, relative_time):
```

Comment:
I'd rename this function to make the distinction of what is doing now.

Reply:
Done.